packages/components/webpack.config.js

```diff
@@ -74,25 +74,25 @@ module.exports = {
                     },
                 ],
             },
-        ( !isServe ? {
-                    enforce: "pre",
-                    test: /\.(js|jsx)$/,
-                    exclude: [/node_modules/],
-                    loader: "eslint-loader",
-                    options: {
-                        fix: true
-                    },
-                } :{}),                
+            (!isServe ? {
+                enforce: 'pre',
+                test: /\.(js|jsx)$/,
+                exclude: [/node_modules/],
+                loader: 'eslint-loader',
+                options: {
+                    fix: true
+                },
+            } : {}),
             {
-                test   : /\.(js|jsx)$/,
+                test: /\.(js|jsx)$/,
                 exclude: /node_modules/,
-                loader : 'babel-loader',
+                loader: 'babel-loader',
             },
         ],
     },
-    plugins: [       
-        new MiniCssExtractPlugin({ filename: 'deriv-components.css' }),
-        new StyleLintPlugin( { fix: true }),
+    plugins: [
+        new MiniCssExtractPlugin({ filename: '[name].css' }),
```

Comment:
We should also add `production` and `development` detection.

Reply:
@reza-fs why we need that?

Reply:
We could leverage the long term caching if we could add something like `[name].[contenthash].css`, AFAIK. Unless we want to defer this to the user of the component. What do you think?

Reply:
@reza-fs as we talked its being handled on user side.